libs/openchallenges/org-search/src/lib/org-search-filters-values.ts

```diff
@@ -0,0 +1,19 @@
+import { FilterValue } from '@sagebionetworks/openchallenges/ui';
+
+export const organizationTypesFilterValues: FilterValue[] = [
```

Comment:
Are `types` the same as `roles`?

Reply:
Yea, the property is not in the `Organization` yet, but I think `roles` will be more appropriate term. 

cc @tschaffter

Reply:
Correct, that would be the contribution role (previously named organization role).